hack/cr.yaml

```diff
@@ -6,15 +6,17 @@ metadata:
 spec:
   managementState: "Managed"
   nodeSpec:
-    image: openshift/origin-logging-elasticsearch5:latest
+    image: docker.io/openshift/origin-logging-elasticsearch5:latest
+    resources:
+      limits:
+        memory: 1Gi
```

Comment:
Understand this is a 'move' but why are we not specifying the values to be the same given this is our general strategy in prior releases to ensure the node is properly scheduled.

Reply:
what do you mean when you're saying they aren't the same? Are you referring to the ansible defaults?